BXFinance_iOS_source/Sample Code/BXFinance/OIDCViewController.swift

```diff
@@ -137,3 +154,31 @@ class OIDCViewController: UIViewController {
         }
     }
 }
+
+extension OIDCViewController: DocumentSubmissionListener {
+    func onDocumentSubmitted(response: DocumentSubmissionResponse) {
+        print("The document status is \(String(describing: response.documentStatus))")
+        print("The document submission status is \(String(describing: response.documentSubmissionStatus))")
+            
+        guard let documents = response.document else { return }
+        for (key, value) in documents {
+            print("\(key): \(value)")
+        }
+        
+        self.verifiedFirstName = (documents["firstName"] ?? "").capitalized
+        self.verifiedLastName = (documents["lastName"] ?? "").capitalized
+    }
+    
+    func onSubmissionComplete(status: DocumentSubmissionStatus) {
+        // present a basic alert to indicate completion
+        let alertController = UIAlertController(title: "Document Submission Complete", message: "All documents for \(self.verifiedFirstName) \(self.verifiedLastName) have been submitted and verification has successfully completed.", preferredStyle: .alert)
```

Comment:
This won't display the correct message if the app is used to run a verify transaction not asking for an ID.